src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const [isHovered, setHover] = useState(false);
```

Comment:
```suggestion
  const [isHovered, setIsHovered] = useState(false);
```
Second variable should be named according to the first variable